.github/workflows/release.yaml

```diff
@@ -284,7 +284,7 @@ jobs:
         run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
       - name: Build  🔧
         run: |
-          earthly +all -VARIANT=standard -K3S_VERSION=${{ matrix.k3s_version }} -FLAVOR=${{ matrix.flavor }}
+          earthly +all -VARIANT=standard -K3S_VERSION="${{ matrix.k3s_version }}" -FLAVOR=${{ matrix.flavor }}
```

Comment:
Locally when building the image, it was failing to find the package and it worked when I added quotes (there is a `+` symbol in the version). But I tend to think it was something irrelevant (earthly caches?). Not sure if the quotes are needed.